src/cpu/aarch64/jit_sve_512_x8s8s32x_conv_kernel.cpp

```diff
@@ -0,0 +1,1481 @@
+        b(oi_loop_label);
+
+        // middle or last ow block entry
+        L(middle_ow_blocks_label);
+
+        if (jcp.l_pad > 0) {
+            // just to consider left padding, not compute
+            adds_imm(
+                    reg_inp, reg_inp, inp_shift_pad_second_block, reg_tmp0_imm);
+        }
+
+        // set number of iteration for oi-loop
+        if (n_oi_last_ow_block != n_oi_not_last_ow_block) {
+            cmp(reg_owb, jcp.nb_ow - 1); // last ow-block ?
+            mov_imm(reg_oi, n_oi_last_ow_block);
+            b(EQ, oi_loop_label);
+        }
+
+        if (n_oi_next_last_ow_block != n_oi_not_last_ow_block) {
+            cmp(reg_owb, jcp.nb_ow - 2); // next to last ow-block ?
+
+            mov_imm(reg_oi, n_oi_next_last_ow_block);
+            b(EQ, oi_loop_label);
+        }
+        mov_imm(reg_oi, n_oi_not_last_ow_block); // other middle ow-blocks
+
+        // oi loop w/o padding
+        L(oi_loop_label);
+        {
+            cmp(reg_oi, 0);
+            b(LE, oi_loop_end_label);
+
+            icb_loop(jcp.ur_w, 0, 0, false);
+
+            adds_imm(reg_inp, reg_inp, inp_shift, reg_tmp0_imm);
+            adds_imm(reg_out, reg_out, out_shift, reg_tmp0_imm);
+            subs(reg_oi, reg_oi, 1);
+
+            b(oi_loop_label);
+        }
+        L(oi_loop_end_label);
+
+        ldr(reg_owb, ptr(reg_param1, GET_OFF(owb)));
+        cmp(reg_owb, 0); // first ow-block ?
+        if (first_ow_block_padded)
+            b(EQ, last_oi_label);
+        else
+            b(EQ, end_label);
+
+        cmp(reg_owb, jcp.nb_ow - 2); // next to last ow-block ?
+        b(LT, end_label);
+        if (next_last_ow_block_padded)
+            b(EQ, last_oi_label);
+        else
+            b(EQ, end_label);
+
+        // that is last block
+        if (!last_ow_block_padded) b(tail_label);
+
+        // last oi block with right padding
+        L(last_oi_label);
+        icb_loop(jcp.ur_w, 0, r_pad1, jcp.ur_w_tail == 0);
+        adds_imm(reg_inp, reg_inp, inp_shift, reg_tmp0_imm);
+        adds_imm(reg_out, reg_out, out_shift, reg_tmp0_imm);
+
+        ldr(reg_owb, ptr(reg_param1, GET_OFF(owb)));
+        cmp(reg_owb, jcp.nb_ow - 1); // last ow_block?
+        b(LT, end_label);
+
+        // ur_w tail
+        L(tail_label);
+        if (jcp.ur_w_tail != 0) { icb_loop(jcp.ur_w_tail, 0, r_pad, true); }
+        L(end_label);
+    }
+    postamble();
+
+    if (jcp.is_fast_depthwise) {
+        align(64);
+        L(permute_index_table);
+        const uint32_t _idx[]
+                = {0, 4, 8, 12, 1, 5, 9, 13, 2, 6, 10, 14, 3, 7, 11, 15};
+        for (size_t i = 0; i < sizeof(_idx) / sizeof(_idx[0]); ++i)
+            dw(_idx[i]);
+    }
+}
+
+bool jit_sve_512_x8s8s32x_fwd_kernel::post_ops_ok(
+        jit_conv_conf_t &jcp, const primitive_attr_t &attr) {
+    using namespace primitive_kind;
+    const auto &p = attr.post_ops_;
+
+    /* At this time, post_op is not supported. */
+    return p.len() ? false : true;
```

Comment:
nit: `return 0 == p.len()`

Reply:
Hi @msotoflo ,

Thanks for giving us a suggestion. We have updated the code as you suggested.